### ISSUE TITLE
Picture uploaded into asset volume not shown

### ISSUE THREAD

**Issue by tomdeleu** [engram-design/CPNav#52]
### Description

It seems any picture (.svg) uploaded in a local asset volume doesn't work. Only files uploaded into "temporary uploads" actually appear in the menu. All other images are just shown as the default "circle with a letter".

In the CMS, the picture is displayed correctly.

I do have other asset volumes defined (Amazon S3) but I'm uploading the icon (.svg) in a second local asset volume.

[View screenshot](https://cl.ly/0h221S333Z1n)

### Steps to reproduce

1. Add new menu item
2. Choose icon and upload in an existing local asset folder

### Additional info

- Plugin version: 2.0.8
- Craft version: 3.0.18

**Comment by dennisfrank** [comment]
I have the same problem.

**Comment by engram-design** [comment]
I wonder if its your specific SVG? I've tried with a number of different ones, and its worked as expected.

For example:
```
<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48"><g fill="#e64c4c"><path d="M24 47C11.317 47 1 36.682 1 24S11.317 1 24 1s23 10.318 23 23-10.317 23-23 23zm0-44C12.421 3 3 12.42 3 24s9.421 21 21 21 21-9.42 21-21S35.579 3 24 3z"/><path d="M24 47a1 1 0 0 1-1-1V2a1 1 0 1 1 2 0v44a1 1 0 0 1-1 1z"/><path d="M43 15H5a1 1 0 1 1 0-2h38a1 1 0 1 1 0 2zM43 35H5a1 1 0 1 1 0-2h38a1 1 0 1 1 0 2z"/><path d="M23.111 47a.999.999 0 0 1-.542-1.841C29.811 40.505 34.134 32.595 34.134 24S29.811 7.495 22.569 2.841a1 1 0 0 1 1.082-1.683C31.468 6.182 36.134 14.721 36.134 24s-4.666 17.818-12.482 22.842a1.005 1.005 0 0 1-.541.158z"/><path d="M24.133 47a.992.992 0 0 1-.54-.159C15.776 41.818 11.11 33.279 11.11 24S15.776 6.182 23.593 1.158a1 1 0 0 1 1.082 1.683C17.434 7.495 13.11 15.405 13.11 24s4.323 16.505 11.564 21.159A1 1 0 0 1 24.133 47z"/></g></svg>
```

**Comment by tomdeleu** [comment]
Still the same issue unfortunately, even with your exact SVG: https://cl.ly/37407a59a090

**Comment by engram-design** [comment]
Interesting. What sort of environment are you on? Latest Craft?

What would be the path of your volume as well? Using any aliases?

**Comment by tomdeleu** [comment]
Yes latest everything, just did an update right before retesting it.

**Comment by engram-design** [comment]
Aha - I believe its when folders are using an alias. Should be fixed in [2.0.9](https://github.com/verbb/cp-nav/releases/tag/2.0.9)
